### ISSUE TITLE
Updated filledDarker/filledLighter Stories

### ISSUE THREAD

**Issue by spmonahan** [microsoft/fluentui#22967]
Components that support the `filledDarker` and `filledLighter` appearance variants need to show these variants against a sufficiently contrasting background in Storybook.

## Changes to Make

1. Use `#8a8a8a` (Grey 54) for the background color for the `filledDarker` and `filledLighter` variants.
2. Add the design guidance to the story that demonstrates variant appearances.

### Design Guidance

> The colors adjacent to the input should have a sufficient contrast. Particularly, the color of input with
          filled darker and lighter styles needs to provide greater than 3 to 1 contrast ratio against the immediate
          surrounding color to pass accessibility requirement.

## Components to update

- [x] Combobox #22996
- [x] Dropdown -- part of Combobox
- [x] Input - #22966
- [x] Select - #23050
- [x] SpinButton - #22980
- [x] Textarea - #22987
